staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -45,20 +64,20 @@ type Transformer interface {
 	// TransformFromStorage may transform the provided data from its underlying storage representation or return an error.
 	// Stale is true if the object on disk is stale and a write to etcd should be issued, even if the contents of the object
 	// have not changed.
-	TransformFromStorage(data []byte, context Context) (out []byte, stale bool, err error)
+	TransformFromStorage(data []byte, context context.Context) (out []byte, stale bool, err error)
 	// TransformToStorage may transform the provided data into the appropriate form in storage or return an error.
-	TransformToStorage(data []byte, context Context) (out []byte, err error)
+	TransformToStorage(data []byte, context context.Context) (out []byte, err error)
```

Comment:
same here (keeping the existing explicit typed parameter, and adding `ctx context.Context` as the first param)